registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleManager.java

```diff
@@ -224,7 +228,15 @@ public void consume(MosipEventBus mosipEventBus, MessageBusAddress fromAddress,
 	}
 
 	public Integer getEventBusPort() {
-		return Integer.parseInt(eventBusPort);
+		return environment.getProperty(getPropertyPrefix() + "eventbus.port", Integer.class);
```

Comment:
Since environment.getProperty() is dynamic, this can be evaluated after the startup only. Creating a PropertiesUtil, in which we will check for null for the property and throw exception accordingly.